ckanext/unhcr/templates/package/read_base.html

```diff
@@ -24,13 +24,14 @@
 {% block content_action %}
   {% if pkg.kobo_asset_id and h.check_access('package_update', {'id':pkg.id }) %}
     <button 
-        id="kobo-pkg-update-resources"
-        class="btn btn-default"
+        class="btn btn-default kobo-pkg-update-resources"
         data-kobo-asset-id="{{ pkg.kobo_asset_id }}"
+        data-force="false"
         data-kobo-update-endpoint="{% url_for 'unhcr_kobo.enqueue_survey_update' %}"
     >
     <i class="fa fa-refresh"></i>
-    Update KoBoToolbox data</button>
+    <span class="update-kobo-button-text">Update KoBoToolbox data</span>
```

Comment:
```suggestion
    <span class="update-kobo-button-text">Update KoBo data</span>
```
(Unless this was required by the client)